packages/ibm-products/src/components/Card/Card.js

```diff
@@ -193,7 +194,7 @@ export let Card = forwardRef(
         className: cx(`${blockClass}__header-body-container`, {
           [`${blockClass}__clickable`]: clickable,
         }),
-        ...(clickable && clickableProps),
+        ...(clickable || (stackedBody && clickableProps)),
```

Comment:
so in this notation it's spreading `clickableProps` if `clickable` is `true`. how you actually want to write this is if either `clickable` or `stackedBody` is `true`. so it should look like
```
...((clickable || stackedBody) && clickableProps)
```
this notation can look confusing if you're unfamiliar with it. i hope this helps clarify!